core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/resource/CommonStatus.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.resource;
+
+public enum CommonStatus {
```

Comment:
ResourceCommonStatus, CommonStatus is too generic

Reply:
This is a copy of a class that already exists. I just copied it here so that datalake can use it when calling to get stack with resources. not changing it to keep it consistent with the existing classes